demisto_sdk/commands/run_cmd/runner.py

```diff
@@ -91,14 +92,21 @@ def _get_playground_id(self):
         """
         playground_filter = {'filter': {'type': [9]}}
         answer = self.client.search_investigations(filter=playground_filter)
-
-        if answer.total != 1:
+        user_data, response, _ = self.client.generic_request(path='/user',
+                                                             method='GET',
+                                                             content_type='application/json',
+                                                             response_type=object)
+        if response != 200:
+            raise RuntimeError('Cannot find username')
+        username = user_data.get('username')
+        playgrounds = [playground for playground in answer.data if playground.creating_user_id == username]
+        if len(playgrounds) != 1:
```

Comment:
Let's put this flow after the `len(plagrounds) != 1`. If there's only one playground, there's no need to do the user filter.